server/util/rexec/rexec.go

```diff
@@ -0,0 +1,282 @@
+// package rexec provides a collection of utility functions for remote execution
+// clients.
```

Comment:
nit: rm "a collection of" and make one line

Reply:
Done